modules/network-observability-netobserv-cli-reference.adoc

```diff
@@ -0,0 +1,190 @@
+// Module included in the following assemblies:
+// * observability/network_observability/netobserv-cli-reference.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-netobserv-cli-reference_{context}"]
+= oc netobserv CLI reference 
+The Network Observability CLI (`oc netobserv`) is a CLI tool for capturing flow data and packet data for further analysis.
+
+.`oc netobserv` syntax
+[source,terminal]
+----
+$ oc netobserv [<command>] [<feature_option>] [<command_options>] <1>
+----
+<1> Feature options can only be used with the `oc netobserv flows` command. They cannot be used with the `oc netobserv packets` command.
+
+[cols="3a,8a",options="header"]
+.Basic commands
+|===
+|Command| Description
+
+| `flows`
+| Capture flows information. For subcommands, see the "Flow capture subcommands" table. 
+
+| `packets`
+| Capture packets from a specific protocol or port pair, such as `netobserv packets --filter=tcp,80`. For more information about packet capture, see the "Packet capture subcommand" table.
+
+| `cleanup`
+| Remove the Network Observability CLI components.
+
+| `version`
+| Print the software version.
+
+| `help`
+| Show help.
+|===
+
+[id="network-observability-cli-enrichment_{context}"]
+== Network Observability enrichment
+The Network Observability enrichment to display zone, node, owner and resource names including optional features about packet drops, DNS latencies and Round-trip time can only be enabled when capturing flows. These do not appear in packet capture pcap output file.
+
+.Network Observability enrichment syntax
+[source,terminal]
+----
+$ oc netobserv flows [<enrichment_options>] [<subcommands>]
+----
+
+.Network Observability enrichment options
```

Comment:
I removed packet drops from this table cc @msherif1234 @Amoghrd Otherwise, there are no mentions of that feature in the CLI doc.

Reply:
For 4.12 only

Reply:
https://github.com/openshift/openshift-docs/pull/77518/files#diff-c18247ebe497a5c92e79ac355fd76da99dc3ee86cd685775eb99c32e11eea0ceR37-R47